GSA/digital-gov-drupal
# DIGITAL-61: export default content type for glossary

## Jira ticket

[DIGITAL-61](https://cm-jira.usa.gov/browse/DIGITAL-61)

##Relates to

[Digital-43](https://github.com/GSA/digital-gov-drupal/pull/75)
[Digital-60](https://github.com/GSA/digital-gov-drupal/pull/78 )

## Purpose

Acceptance Criteria:

A few sample entities are exported via default content.

### QA/Testing instructions

1. Pull branch
2. Drush CIM
3. lando rebuild && lando si
4. Taxonomy module should be enabled.

1. Edit one of the guide pieces of content.
2. Select "HCD Glossary" under the HCD Glossary field.
3. Make sure the "Glossary" button appears.

## Checklist for the Developer


## Checklist for the Peer Reviewers
